### ISSUE TITLE
 Browse mode does not work and user can't consume some content in Settings 

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#15963]
### Steps to reproduce:

1. Go to Settings App, any screen with NVDA On
2. Press Caps Space or Insert Space to switch to browse mode. Observe
3. Go to Language and Region settings in Windows
4. Tab to Regional format section and activate the chevron to expand it
5. Try to switch to NVDA Browse mode to consume this content. Observe.

### Actual behavior:
NVDA users are unable to switch to browse mode. There by cant consume some content in Settings screens.

### Expected behavior:
NVDA users should be able to switch to browse mode in Settings App too.

### NVDA logs, crash dumps and other attachments:
[NVDA_BrowseMode_DoesNotWork](https://microsoft-my.sharepoint.com/:v:/p/v-bmoogala/Ed55oyRVA2pEuh1reCHox3cBRmhdVgGawDBSL38fDeCyfw?e=36cLJc)
[NVDA_LanguageSettings_BrowseMode](https://microsoft-my.sharepoint.com/:v:/p/v-bmoogala/ESzSg_Y2TqlFs-rG88a2lGcBDBfLujsyOs1Bu8OOcW-Lcg?e=b2RqQ9)

**Comment by josephsl** [comment]
Hi, NVDA sers can use object navigation to consume Settings app contents as browse mode is not really a thing in this app (I think this applies to a vast majority of XAML controls apart from a few that embed web view controls). Thanks.

**Comment by seanbudd** [comment]
this is by design - we do not support browse mode in settings
